src/CoreGraphics/CGContext.cs

```diff
@@ -579,6 +579,16 @@ public void EOClip ()
 		}
 
 		[DllImport (Constants.CoreGraphicsLibrary)]
+		[iOS (11,0), Mac(10,3), TV(11,0), Watch(4,0)]
+		extern static void CGContextResetClip (/* CGContextRef */ IntPtr c);
+
+		[iOS (11,0)]
```

Comment:
`[iOS (11,0), Mac(10,3), TV(11,0), Watch(4,0)]` should match p/invoke
or developers won't have the correct information displayed in the IDE